assets/js/app.js

```diff
@@ -0,0 +1,431 @@
+"use strict";
```

Comment:
considering the size of this `js` file, you were able to keep it clear and understandable, really impressive